src/db/migrations/44-lens-approvals.ts

```diff
@@ -0,0 +1,69 @@
+import { Knex } from 'knex';
+
+import { formatAddress } from '../../types/address';
+import { ChainId, getFactoryId } from '../../types/chain';
+import { CrdtOperation } from '../../types/message';
+import { Table } from '../db';
+
+const lensCurated = [
+  '0x727Ce17C09a74292Eb1594f70e8f048402b6DC02',
```

Comment:
i see 39 tracks here but a bunch less on the preview @ https://spindexer-qa-build.spinamp.xyz/platform/lens

are some broken?

Reply:
Just had to remove a few duplicate values